app/models/build/utils.rb

```diff
@@ -52,7 +52,18 @@ def fix_version_string(version)
         semVerReg = /#{Regexp.escape(specifier)}\s*(\d+\.)*[x\*]/
 
         version.gsub!(semVerReg) do |match|
-          match.strip[0..-3]
+          new = match.strip[0..-3]
+
+          case specifier
+          when '>'
+            new[-1] = (new[-1].to_i + 1).to_s
+            new.gsub specifier, '>='
+          when '<='
+            new[-1] = (new[-1].to_i + 1).to_s
+            new.gsub specifier, '<'
+          else
+            new
+          end
```

Comment:
How does this behave for cases where the previous version segment is multiple digits (e.g. `<= 2.99.x`)?